solana/solana-ibc/programs/solana-ibc/src/lib.rs

```diff
@@ -146,15 +142,6 @@ pub mod solana_ibc {
                 .map_err(error::Error::RouterError)
```

Comment:
There’s no longer need to clone message in line above.

Reply:
right right, got it.